includes/Admin/Admin_Page.php

```diff
@@ -164,11 +164,19 @@ public function render_page() {
 	 *
 	 * @since n.e.x.t
 	 *
+	 * @global string $status
+	 *
 	 * @param array  $actions     List of actions.
 	 * @param string $plugin_file Plugin main file.
 	 * @return array The modified list of actions.
 	 */
 	public function filter_plugin_action_links( $actions, $plugin_file ) {
```

Comment:
The [`plugin_action_links`](https://developer.wordpress.org/reference/hooks/plugin_action_links/) filter has two additional parameters that both seem to provide the status information. I'd suggest using one of those instead of a global variable.

The third param, `$plugin_data`, is the complete data array for each plugin and has a `status` entry.

The fourth param, `$context`, just provides that same information.

Reply:
Thank you, @swissspidy, for your valuable feedback. I took into consideration the use of the `$status` global variable in various places within the WordPress core. However, I have now updated the PR to incorporate your suggestion. The PR now utilizes the filter parameter as recommended.